deploy/olm-catalog/ibm-iam-operator/3.12.1/ibm-iam-operator.v3.12.1.clusterserviceversion.yaml

```diff
@@ -595,27 +595,27 @@ metadata:
 spec:
   relatedImages:
     - name: ibm-iam-operator
-      image: quay.io/opencloudio/ibm-iam-operator:3.12.1
+      image: quay.io/opencloudio/ibm-iam-operator@sha256:336efcd1aa469c6d23c6c3c51d6a051c0cb0f1c917c9b1c544c3712038a0d550
```

Comment:
Use the tags, but use the *_IMAGE convention. Details provided in the tracking ticket.

Reply:
updated the same